portfolio/src/main/webapp/index.html

```diff
@@ -22,7 +23,8 @@ <h1>Jenny Baik</h1>
 
       <section>
         <div  id="contact-section">
-          <p>temporary text here, probably for comment section</p>
+          <button onclick="getJsonData()">clickme</button>
```

Comment:
instead of this; how about calling getJsonData() in your DOMContentLoaded listener? or window.onload?